packages/keycodes/src/index.js

```diff
@@ -151,7 +151,7 @@ export { isAppleOS };
 /**
  * Map the values of an object with a specified callback and return the result object.
  *
- * @template T
+ * @template {{ [s: string]: any; } | ArrayLike<any>} T
```

Comment:
Related to https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-8.html#unconstrained-generics-no-longer-assignable-to-